src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -353,3 +353,35 @@ databaseChangeLog:
                   name: closed_issue_count
                   type: bigint
                   remarks: The number of existing issues closed by this request (if it was successfully processed at all).
+  - changeSet:
+      id: users-table
+      author: tim.best@usds.dhs.gov
+      comment: information about the users of the case-issue-api
+      changes:
+        - createTable:
+            tableName: user_information
+            remarks: Data about the users that login to the api.
+            columns:
+              - column: *pk_column
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
+              - column:
+                  name: print_name
+                  type: *string
+                  remarks: The human-readable name of this entity (may contain spaces and punctuation).
```

Comment:
"of this user", probably? (I'd say "person" but users may not be people, once we get x509 working.)